src/xenia/base/bit_field.h

```diff
@@ -29,7 +34,7 @@ struct bf {
   // For enum values, we strip them down to an underlying type.
   typedef
       typename std::conditional<std::is_enum<T>::value, std::underlying_type<T>,
-                                std::identity<T>>::type::type value_type;
+                                xe::identity<T>>::type::type value_type;
```

Comment:
std::identity is not in a standard. It was present in some pre c++11 drafts, but was removed later. So now it's specific to VS compiler.